server/build/webpack.js

```diff
@@ -75,8 +75,17 @@ export default async function createCompiler (dir, { hotReload = false, dev = fa
     )
   }
 
-  const babelRuntimePath = require.resolve('babel-runtime/package')
-  .replace(/[\\/]package\.json$/, '')
+  const resolveModule = (moduleName) => {
+    // Check if there's a local version of the module
+    try {
+      const path = `next/node_modules/${moduleName}`
```

Comment:
Perhaps if I write it like this:

```js
const resolveModule = (moduleName) => {
  const path = require.resolve(moduleName)
  const regex = /.*?\/node_modules\/(.*)$/i
  const resolvedPath = path.match(regex)[1]
  return resolvedPath
}
```

This has the exact same behaviour, but since it doesn't explicitly mention `next`, you might think it's less weird? It's definitely intentional to either implicitly (this way) or explicitly (previous way) refer to `next` here.